assets/sprites.json

```diff
@@ -120,6 +120,39 @@
 			"slime_idle_1"
 		]
 	},
+	{
```

Comment:
Waarom eigenlijk een json bestand ipv gewoon rechtstreeks in js? :)

Reply:
Ik vond het goed voelen om de sprite index zelf als een stukje rauwe data op te slaan. Ik heb ideeën om later nog een soort sprite management tooltje te bouwen om sprites en frames definities toe te wijzen. Dat tooltje zou dan JSON outputten.